### PR TITLE
Add example policies, UI policy walkthrough, update config

### LINKED ISSUE #116: Document more Enforce for k8s policy examples
Providing customer with more policy examples is something high on our priority list for Q4. I believe we do have examples now for policies but have yet to document them in the Enforce product docs. There are example policies [here](https://github.com/chainguard-dev/gke-demo/tree/main/policies) and also upstream [here](https://github.com/sigstore/policy-controller/tree/main/examples). Here's [another related doc](https://docs.google.com/document/d/1KzPf53aPANofkBUK0_uD65E4z_i9mUuJyCgBVxQqBNI/edit) we started on curated policy ideas for reference.

### PR DESCRIPTION
**Pull request by ltagliaferri**
Resolves #116 and #153

Also updating config to less frequent crawling cadence.

Signed-off-by: Lisa Tagliaferri

### CODE REVIEW

File: config/_default/config.toml [hunk 1]
```diff
@@ -55,7 +55,7 @@ rel  = "sitemap"
     maxAge = "10s"
 
 [sitemap]
-  changefreq = "hourly"
```
Comment (ltagliaferri) [inline config/_default/config.toml:58]: made this less frequent as we are no expecting hourly changes right now

File: content/ui/enforce-onboarding-quickstart.md [hunk 2]
```diff
@@ -26,16 +26,10 @@ To install `chainctl`, we’ll use the `curl` command to pull the application do
 curl -o chainctl "https://dl.enforce.dev/chainctl_$(uname -s)_$(uname -m)"
```
Comment (ltagliaferri) [inline content/ui/enforce-onboarding-quickstart.md:26]: @katjuell I made a number of updates to the quickstart guide based on changes to the onboarding doc

File: content/ui/enforce-onboarding-quickstart.md [hunk 3]
```diff
@@ -243,4 +240,13 @@ chainctl cluster uninstall
 kind delete cluster --name enforce-demo
 ```
 
-To learn more about Chainguard Enforce, please review our documentation and other resources on Chainguard Academy.
+## Learn more
```
Comment (ltagliaferri) [inline content/ui/enforce-onboarding-quickstart.md:243]: @amdawson — I added a number of links to the UI doc for learning more about Chainguard Enforce. Let me know if there are others we should highlight right now. I think once I know what changes you want to the arch diagram we can add that link too

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/how-to-create-policies-chainguard-enforce-console/index.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+At the top of the page will be three icons you can click on: **Clusters**, **Policies**, and **Packages**. Click on **Policies** to pull up the policies you have available on Chainguard Enforce.
+
+## Inspect any existing policies
+
+If you were invited to a group, you may already have policies available. From the Policy landing page, you can review any existing policies and any existing violations of those poicies. 
+
+![Chainguard Enforce policies page](policies-page.png)
+
+These are displayed in a table by the policy's **Name**, **Violations** and **Groups**, similar to the screenshot below.
```
Comment (SharpRake) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/how-to-create-policies-chainguard-enforce-console/index.md:36]: I'd change the final part of this sentence to ", similar to the following screenshot"

**Review by SharpRake — COMMENTED** [review]
Just one minor comment, otherwise I think this looks good

**Review by SharpRake — COMMENTED** [review]
Just the one comment about getting rid of "below". Otherwise all these changes look good to me.

**Pull request merged**